scripts/create-clusters.sh

```diff
@@ -89,7 +90,7 @@ function create-clusters() {
 
   for i in $(seq ${num_clusters}); do
     # kind will create cluster with name: kind-${i}
-    kind create cluster --name ${i}
+    kind create cluster --name kind-${i}
```

Comment:
i'm not sure why this is necessary. just to match with the comment?

Reply:
```
➜  federation-v2 git:(master) ✗ kind version
0.2.0-alpha
```
My kind create clusters without kind- prefix automatically.